src/Command/CreateModuleCommand.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-mvc for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-mvc/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-mvc/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\Command;
+
+use DirectoryIterator;
+use InvalidArgumentException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+use function copy;
+use function file_get_contents;
+use function file_put_contents;
+use function getcwd;
+use function is_dir;
+use function is_file;
+use function is_writable;
+use function preg_match;
+
+use const DIRECTORY_SEPARATOR;
+
+final class CreateModuleCommand extends Command
+{
+    protected static $defaultName = 'mvc:module:create';
+
+    protected function configure(): void
+    {
+        $this->setDescription('Creates new MVC Module');
+
+        $this->addOption(
+            'dir',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'Directory with modules'
+        );
+
+        $this->addOption(
+            'name',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'New module name'
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $dir = $this->getModuleDir($input, $output);
+        $input->setOption('dir', $dir);
```

Comment:
This is the only thing we have to do, so the input mapper can get the value and pass it to the next command.

I am thinking about helper for that kind of thing, as I'd like all commands behave the same way - check if there is an option provided, and if it is not, ask a question. The answer should be stored back in the input, so it can be passed to the next command (if there is a chain).

Reply:
I think that's a good idea.